guides/meetup.rst

```diff
@@ -323,10 +326,60 @@ Při sdílení myslete na to, že ne každý se rád fotí a ne každý rád vis
 Videa
 ^^^^^
 
-Pokud se vám povedlo natočit nějaká videa (viz :ref:`srazy-nataceni`), tak dejte vědět `Petrovi Viktorinovi <http://encukou.cz/>`_ a on vám řekne, co s nimi. Až ho to nebude bavit, tak tento odstavec přepíše a bude tady přímo návod na to, jak je můžete zpracovat a jak se můžou objevit na `YouTube kanálu Pyvce <https://www.youtube.com/user/pyvec/playlists>`_.
+Pokud se ti povedlo natočit nějaká videa (viz :ref:`srazy-nataceni`), buď dej vědět `Petrovi Viktorinovi <http://encukou.cz/>`_ a nebo se pokus o jejich zpracování sám/sama.
+
+ 1. Připrav prázdný adresář pro každou přednášku. S ti tím pomůže funkce ``videometadata`` nástroje `pyvodb <https://github.com/pyvec/pyvodb>`_:
+
+.. code-block:: shell
+
+ $ python3 -m venv venv
```

Comment:
Nejsem si jistý, zda vše za `.. code-block:: shell` nemá být odsazeno o 4 mezery. Ale možná to řeší ty dvě magické tečky na konci?

Reply:
Tohle jsem řešil. Když tam bylo víc mezer, tak se z nějakého důvodu těch víc mezer dostávalo i do výstupu. V každém případě takhle to vypadá správně po kompilaci.